unit/ctest_null_comm.c

```diff
@@ -79,9 +79,68 @@ test_1d()
     }
   }
 
+  ///////// Test gkyl_comm_array_allgather //////////////
```

Comment:
Please reformat this comment to remove //////// since we don't do that elsewhere.

Also, instead of lengthening this test, can you please create new tests for these methods? you could potentially keep allgather/allgather_host in the same test (same with bcast).

Reply:
I thought this was the right thing to do when writing the test, but this is how the test was organized to begin with. I'll reorganize this